src/components/LinkSyfomodiaperson.tsx

```diff
@@ -94,3 +100,46 @@ export function LinkSyfomodiaperson({
     </div>
   );
 }
+
+interface LinkSyfomodiapersonSideProps {
```

Comment:
Trenger vi alt før `Props`? 😊
```suggestion
interface Props {
```

Reply:
Krasjer litt med eksisterende definisjon som heter `Props` på [linje 66](https://github.com/navikt/syfooversikt/pull/633/files#diff-9ea46e610a0c11338b86ba6147e94bc2a276be4cb1d3d35c71c2481f46795633R66)

Reply:
Aha, her er det 2 komponenter som blir eksportert ja. Lurer på om det ikke er best practice 🤔 [Airbnb har en styleguide for react som sier noe om det](https://arc.net/l/quote/asbmohpi). Ikke at vi må følge airbnb sin styleguide på react, men det gir kanskje en pekepinn på hva som er vurdert best practice og ikke 😊

Reply:
Slik jeg forstår det har en komponent state om den bruker noen form for hooks, og jeg ser at begge komponentene her bruker hooken `useAktivBruker`. Hooks er en form for å koble seg på state, så om komponenten bruker en hook så har den state (slik jeg har forstått det) 😊